src/controller/event/handler/internal/artifact.go

```diff
@@ -55,43 +99,137 @@ func (a *Handler) IsStateful() bool {
 }
 
 func (a *Handler) onPull(ctx context.Context, event *event.ArtifactEvent) error {
+	// if duration is equal to 0 or negative, keep original sync mode.
+	if asyncFlushDuration <= 0 {
+		var tagName string
+		if len(event.Tags) > 0 {
+			tagName = event.Tags[0]
+		}
+
+		if !config.PullTimeUpdateDisable(ctx) {
```

Comment:
do we still have to keep PullTimeUpdateDisable & PullCountUpdateDisable?

cc @stonezdj

Reply:
I hope not.

Reply:
use other pr to remove it if we need.

Reply:
We should keep the setting of PullAuditLogDisable